stock_available/models/product_template.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2014 Numérigraphe
+# Copyright 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api
+from odoo.addons import decimal_precision as dp
+import operator as py_operator
+
+OPERATORS = {
```

Comment:
This is used more than once... perhaps can be good export it to a separated .py ? for example "utils.py"